packages/amplify_datastore/CHANGELOG.md

```diff
@@ -9,6 +9,17 @@
 
   NOTE: Data stored in local database and not synced to cloud will be lost, as [local migration is not supported](https://docs.amplify.aws/lib/datastore/schema-updates/q/platform/flutter/#local-migrations).
 
+- Adding CustomType functionality introduces a breaking change to the amplify-codegen generated `ModelProvider.dart`, as a new class member `customTypeSchemas` has been added to `ModelProviderInterface.dart`.
+
+  **How to migrate?**
```

Comment:
This is necessary even if they aren't using custom types, correct? I just wanted to confirm.

Reply:
Yes that's correct.